packages/theme-check-node/src/temp.ts

```diff
@@ -0,0 +1 @@
+export type AbsolutePath = string;
```

Comment:
Is this file created on accident? I think you wanted to place this code in `path.ts` since that file is empty (and probably wanted to delete this file)

Reply:
absolutely correct haha. That's a leftover from the refactor.